src/locales/it.yaml

```diff
@@ -0,0 +1,360 @@
+      z_adjust_values: Valori regolazione Z
+      gcode_coords: Usa Coordinate GCode
+      time_estimates: Tempo stimato
+      invert_z_control: Inverti Z
+      invert_y_control: Inverti Y
+      invert_x_control: Inverti X
+      enable: Attiva
+      fps_target: Target FPS
+    btn:
+      add_camera: Aggiungi Webcam
+      select_theme: Seleziona Tema
+      add_thermal_preset: Aggiungi Profilo
+      reset: Ripristina
+    timer_options:
+      duration: Solo Durata
+      duration_description: Simile all'LCD di klipper, mostra solo la durata senza stime.
+      file: Stima da File
+      file_description: >-
+        Usa la percentuale di progresso e la durata per stimare la durata
+        totale.<br /> Diventa più preciso con il tempo.
+      filament: Stima dal Filamento
+      filament_description: >-
+        Usa il conteggio del filamento usato e lo compara con il totale stimato
+        di filamento<br/> necessario per stimare la durata totale.<br/> Diventa
+        più preciso con il tempo.
+      slicer: Slicer
+      slicer_description: Usa la stima dello slicer. Devi attivarla nel tuo slicer.
+    camera_type_options:
+      video: IP Webcam
+      mjpegstream: MJPEG Stream
+      mjpegadaptive: MJPEG Adaptive
+    tooltip:
+      gcode_coords: >-
+        Usa il posizionamento GCode invece del posizionamento della testina nel
+        cruscotto.
+  chart:
+    tooltip:
+      help: >-
+        Tieni premuto shift per zommare.<br /> Clicca un elemento da mostrare /
+        nascondere sul grafico.<br /> Clicca una potenza da mostrare /
+        nascondere sul grafico.
+    label:
+      item: Elemento
+      current: Attuale
+      target: Target
+      power: Potenza
+  file_system:
+    title:
+      rename_file: Rinomina File
+      rename_dir: Rinomina Cartella
+      add_file: Aggiungi File
+      add_dir: Aggiungi Cartella
+      download_file: Recupero file
+      upload_file: Caricando il File | Caricando i File
+    label:
+      dir_name: Nome Cartella
+      file_name: Nome File
+      disk_usage: Uso del Disco
+      downloaded: Scaricati
+      transfer_rate: Velocità Trasferimento
+      uploaded: Caricato
+    msg:
+      not_found: Nessun File Trovato
+      processing: Processsando...
+      confirm: Sei sicuro? Questo eliminerà tutti i file e le cartelle all'interno.
+    tooltip:
+      disabled: I Lavori sono disattivati prima della comunicazione iniziale con klippy
+      low_on_space: Poco Spazio su Disco
+    overlay:
+      label: <strong>Trascina</strong> un file qua.
+    filters:
+      label:
+        print_start_time: Filtra stampati
+        print_start_time_desc: Filtra elementi che hai già stampato.
+  endstop:
+    label:
+      open: LIBERO
+      triggered: ATTIVATO
+    msg:
+      subtitle: Usa il pulsante di ricarica per aggiornare lo stato degli endstop.
+  history:
+    msg:
+      confirm: >-
+        Sei Sicuro? Cancellerà tutta la cronologia, e le statistiche della
```

Comment:
I think this should be: `Sei Sicuro? Questa azione cancellerà tutta la cronologia [...]`